File: ios/Classes/OpenPreferenceCenterEvent.swift
/* Copyright Airship and Contributors */

import Foundation

struct OpenPreferenceCenterEvent: TiEvent {
    let name: String = EventNames.openPreferenceCenter
    let data: [String: Any]
    init(preferenceCenterID: String) {
        self.data = ["preferenceCenterId": preferenceCenterID]
    }
}

File: android/src/ti/airship/events/OpenPreferenceCenterEvent.kt
/* Copyright Airship and Contributors */

package ti.airship.events

import ti.airship.TiAirshipModule

class OpenPreferenceCenterEvent(preferenceCenterId: String) : Event {
    override val data: Map<String, Any> = mapOf("preferenceCenterId" to preferenceCenterId)
    override val name: String = TiAirshipModule.eventOpenPreferenceCenter
}